Sources/Xcodeproj/XcodeProjectModelSerialization.swift

```diff
@@ -470,19 +470,19 @@ fileprivate class PropertyListSerializer {
     /// dictionary is what gets written to the property list.
     var idsToDicts = [String: PropertyList]()
 
-    /// Returns the identifier for the object, assigning one if needed.
+    /// Returns the quoted identifier for the object, assigning one if needed.
     func id(of object: PropertyListSerializable) -> String {
         // We need a "serialized object ref" wrapper for the `objsToIds` map.
         let serObjRef = SerializedObjectRef(object)
         if let id = objsToIds[serObjRef] {
-            return id
+            return "\"\(id)\""
```

Comment:
Yeah, agreed. I hope we don't allow either of those in package or targets names, which is the only way that would come up, so I can live with this restriction for now.